packages/popover/components/PopoverListItemAvatar.tsx

```diff
@@ -1,20 +1,8 @@
 import * as React from "react";
 import Avatar, { AvatarProps } from "../../avatar/components/Avatar";
 
-interface PopoverListItemAvatarProps extends AvatarProps {
-  /**
-   * Which side of the menu item the avatar appears on
-   */
-  position?: "start" | "end";
-}
```

Comment:
Was there a reason you removed this interface?

It looks like we still want this here. The component will use the `position` prop which does not exist on `AvatarProps`. 

To maintain backward compatibility with the `defaultProps`, we'll want to make sure we set a matching fallback for position `position = "start"`

Reply:
Hey @nataliepina, thanks for mentioning this. I removed this interface as I was getting a TypeScript error when passing the `position` prop like this as `position` was not a defined prop for `Avatar`:
```js
const PopoverListItemAvatar = ({
  size = "xs",
  position = "start",
  ...props
}: PopoverListItemAvatarProps) => (
  <Avatar size={size} position={position} {...props} />
);
```

The error was:
```
Property 'position' does not exist on type 'IntrinsicAttributes & AvatarProps'
```
I had assumed the prop `position` was not required to pass here since `Avatar` didn't seem to use it, but apologies for my incorrect assumption 😬

Since the `Avatar` component doesn't have a `position` prop defined, how should this be handled? Should I include it as part of the props spread?

Reply:
No worries, I can totally see where you'd be coming from there, and trying to clean it up as it doesn't initially seem like it's used with the props spread and encountering that error.

It also can be tricky if you don't know these components well as some components are used by others. The `PopoverListItemAvatar` and `PopoverListItemIcon` are used in our `DropdownMenu` component. This is where it could take in a start or end value for the position.
Check out [this component story](PopoverListItemAvatar ) and under the Docs tab if you click "Code" you'll see the example. 
Here is a screenshot of that:
<img width="820" alt="Screenshot 2023-03-08 at 9 57 10 AM" src="https://user-images.githubusercontent.com/34781875/223764182-c7bc8ce3-3db1-4158-8357-f9167e64dacc.png">

I think as you suggested you could let the props spread handle it as a simple solution. We may want to consider adding position to the prop types for Avatar and Icon in the future but I'm sure we're there yet.

Reply:
No problem! Once you update this and pull in the changes from main it's a ✅ for me! Thank you for all of your effort on this update.

Reply:
Hey @nataliepina, I've run into an issue when testing these components using default values instead of `defaultProps`. I noticed that if no `position` prop was passed down to `PopoverListItemAvatar` or `PopoverListItemIcon`, the following variable `itemGraphicStart` in `PopoverListItem` is undefined:
```js
      const itemGraphicStart = itemChildren.find(
        child => isItemGraphic(child) && child.props.position === "start"
      );
```

However if I add a `defaultProps` value for `position`, `itemGraphicStart` correctly finds the item and it works as expected. Would you have some guidance on what could be causing this issue? Is it because React doesn't know what the default values would be but it can determine the `defaultProps`?

I did play around with it a little bit and I found that changing `itemGraphicStart` to the following resolved the issue:

```js
      const itemGraphicStart = itemChildren.find(
        child =>
          isItemGraphic(child) &&
          (child.props.position === "start" || !child.props.position)
      );
```

Let me know your thoughts on this!

Reply:
This is a pretty tricky one with those functions to set `itemGraphicStart` and `itemGraphicEnd`. I've tested some different methods to avoid having to update these functions, but I think the implementation you suggested will be fine and it does work. Great detective work on this.